includes/wc-deprecated-functions.php

```diff
@@ -370,8 +370,8 @@ function woocommerce_paying_customer( $order_id ) {
  * @deprecated 3.0
  */
 function woocommerce_customer_bought_product( $customer_email, $user_id, $product_id ) {
-	wc_deprecated_function( __FUNCTION__, '3.0', 'wc_customer_bought_product' );
-	return wc_customer_bought_product( $customer_email, $user_id, $product_id );
+	wc_deprecated_function( __FUNCTION__, '3.0', 'WC_Order_Product_Lookup_Helper::is_product_purchased' );
```

Comment:
I don't know if it's wise to deprecate these, given `WC_Order_Product_Lookup_Helper::is_product_purchased` is a temporary solution.

Ping @kloon @mikejolley